CONTRIBUTING.md

```diff
@@ -188,15 +188,19 @@ As subject matter experts, *code owners* approve pull requests on the packages t
 
 Acceptance of contributions follows the consensus-seeking process.
 
-All pull requests must be approved by a *collaborator* before the pull request can be accepted.
+All pull requests must be approved by an assigned *collaborator* before the pull request can be accepted. A *collaborator* will be assigned as the designated reviewer of a pull request within [a reasonable period of time](#response-time) of the pull request creation. Other users may provide reviews, however the *collaborator* assigned to the pull request will have final say on approval.
 
-Before a pull request is accepted, time should be given to receive input from *collaborators* or *code owners* with the expertise to evaluate the changes. The amount of time can vary but at least 3 days during the typical working week and 5 days over weekends should be given to account for international time differences and work schedules.
+Before a pull request is accepted, [a reasonable period of time](#response-time) should be given to receive input from *collaborators* or *code owners* with the expertise to evaluate the changes.
```

Comment:
As it relates to the last section, is the expectation that all PR's have been reviewed by all stakeholders _within_ 3 days?

Reply:
I believe the response time should be the same for any feedback that's expected to be submitted. If something is too contentious we could have a different process such as opening a discussion.